src/layout_components.py

```diff
@@ -336,26 +344,60 @@ def get_tab_section():
     )
     return tab_section
 
+# Collapse (integrated into navbar below)--
+collapse = dbc.Row(
+     dbc.Button(
+            "About",
+            id="collapse-button",
+            className="mb-3",
+            outline=False,
+            style={'margin-top': '10px',
+                'width': '150px',
+                'background-color': '#0B6623',
+                'color': 'white'}
+        ), justify='end'
+)
+
+collapse_bar = dbc.Row([
+    dbc.Col(
+        dbc.Collapse(
+            html.P([
+                    """This dataset is derived from the spotifyr package and contains over 20,000 songs, 10,000 artists 
+                    and 6 main music genres on Spotify.""", html.Br(),  
```

Comment:
```suggestion
                    and 6 main music genres on Spotify released up until 2020.""", html.Br(),  
```
Small suggestion to mention date of data collection (either 2020 or 2020-01-21)

Reply:
Sure! Will do that